src/main/telemetry/msp_shared.c

```diff
@@ -271,13 +271,14 @@ bool handleMspFrame(uint8_t *const payload, uint8_t const payloadLength, uint8_t
 bool sendMspReply(const uint8_t payloadSizeMax, mspResponseFnPtr responseFn)
 {
     static uint8_t seq = 0;
+    static bool headerSent = false;   
 
     uint8_t payloadArray[payloadSizeMax];
     sbuf_t payloadBufStruct;
     sbuf_t *payloadBuf = sbufInit(&payloadBufStruct, payloadArray, payloadArray + payloadSizeMax);
 
     // detect first reply packet
-    if (responsePacket.buf.ptr == responseBuffer) {
+    if (!headerSent) {
```

Comment:
Another option is to use `responsePacket.buf.ptr == responseBuffer && !skipHeader` and set skipHeader = `chunkRemainder == 0` (in previous iteration). But that is probably overcomplicated.